src/main/java/io/numaproj/numaflow/sourcer/AckRequestImpl.java

```diff
@@ -0,0 +1,15 @@
+package io.numaproj.numaflow.sourcer;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+/**
+ * AckRequestImpl is the implementation of AckRequest.
+ */
+@Getter
```

Comment:
Is it best practice to use the Lombok-generated method as interface impl just because the name of the overridden method starts with get?

Reply:
not sure if it is the best practice, removed it to make it easier to read.